cvat/apps/iam/rules/tests/configs/projects.csv

```diff
@@ -1,10 +1,10 @@
 Scope,Resource,Context,Ownership,Limit,Method,URL,Privilege,Membership
-create,Project,Sandbox,N/A,resource['user']['num_resources'] < 3,POST,/projects,User,N/A
-create,Project,Organization,N/A,resource['user']['num_resources'] < 3,POST,/projects,User,Supervisor
+create,Project,Sandbox,N/A,,POST,/projects,User,N/A
+create,Project,Organization,N/A,,POST,/projects,User,Supervisor
 create,Project,Sandbox,N/A,,POST,/projects,Business,N/A
 create,Project,Organization,N/A,,POST,/projects,Business,Supervisor
```

Comment:
Now these rules for Business and for User are the same and so I think we should remove such repetition 
```suggestion
create,Project,Sandbox,N/A,,POST,/projects,User,N/A
create,Project,Organization,N/A,,POST,/projects,User,Supervisor
```
The same comment for all others csv files that have been changed in this PR

Reply:
I'm not sure, the User and Business are still different privileges, aren't they?

Reply:
Yes, User and Business are different privileges, but these .csv files built by principle: each line means that a certain action is allowed for a user with certain privileges, as well as for all other users with higher privileges. 

And so if it was already mentioned that create,Project,Sandbox is allowed for User, this automatically means that this action is also allowed for Business. We also use this fact during test generation.

Reply:
Ok, it makes sense then. I'll update the files.

Reply:
Done